src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
 
-export class Circle {
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0
+      || b <= 0
+      || c <= 0
+      || a + b <= c
+      || a + c <= b
+      || b + c <= a
+    ) {
+      throw new Error('Sides should be positive and form a valid triangle.');
+    }
 
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.floor(area * 100) / 100;
+  }
 }
 
-export class Rectangle {
+export class Circle implements Figure {
+  shape = 'circle';
+
+  color: string;
+
+  radius: number;
+
+  constructor(color: string, radius: number) {
+    if (radius <= 0) {
+      throw new Error('Radius should be a positive number.');
+    }
 
+    this.color = color;
+    this.radius = radius;
+  }
+
+  getArea(): number {
+    const area = Math.PI * this.radius ** 2;
+
+    return Math.floor(area * 100) / 100;
+  }
 }
 
-export function getInfo(figure) {
+export class Rectangle implements Figure {
+  shape = 'rectangle';
+
+  color: string;
+
+  width: number;
+
+  height: number;
+
+  constructor(color: string, width: number, height: number) {
+    if (width <= 0 || height <= 0) {
+      throw new Error('Width and height should be positive numbers.');
+    }
+
+    this.color = color;
+    this.width = width;
+    this.height = height;
```

Comment:
can be less 
```suggestion
  constructor(
   private color: string,
   private width: number, 
   private height: number
  ) {
    if (width <= 0 || height <= 0) {
      throw new Error('Width and height should be positive numbers.');
    }
```
check other places